hub-templates/base-hub/values.yaml

```diff
@@ -75,20 +87,9 @@ jupyterhub:
         limits:
           memory: 1Gi
     https:
+      enabled: true
```

Comment:
Would this deploy the autohttps pod? We don't need that, right? I think the letsencrypt section here can also be removed, since we use nginx-ingress + cert-manager for it.

Reply:
I think it would yes because letsencrypt.contactEmail is also specified and the default https type is letsencrypt.

With nginx-ingress + cert-manager, set this to false I think.